### DISCUSSION TITLE
Fluent API timeline

### DISCUSSION THREAD

**Discussion by pojacks** [qos-ch/slf4j#288] [Q&A] [answered]
I'd like to use the fluent API of the 2.0.0 release, and I wonder if it's considered safe to use the alpha 7 version, or that there is a dangerous apis in there?

I'd also like to know about timelines for the GA release

**Comment by ceki** [comment, marked as answer]
The alpha qualifier designates the fact that the fluent API might change. However, any breaking changes between alpha-N and alpha-N+1 is likely to be done in a backward compatible manner.

**Reply by chpatel3** [reply]
We are also looking to upgrade to 2.0.0 and wondering what is timeline for GA release? Do you have specific date as we can not depend on alpha version and based on the given GA date we can decide whether to keep using 1.7.x or can upgrade to 2.0.x. Thanks!

**Reply by ceki** [reply]
It should go quickly once the fluent API solidifies.

**Reply by Kawachen97** [reply]
Is there already a date when the discussion about the fluent API will be closed? We also would like to upgrade to the newer version to use the new structured logging.
